core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/user/workflow/WorkflowResource.scala

```diff
@@ -263,22 +259,16 @@ class WorkflowResource {
   @GET
   @Path("/owners")
```

Comment:
similar to this url, `owners` is not very meaningful.

Reply:
Renamed.